### ISSUE TITLE
Bug Report: You cannot access the cpp package while targeting js

### ISSUE THREAD

**Issue by Yuuki795** [KadeArchive/Kade-Engine#2213]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.
Linux
## What version of the game are you using? Look in the bottom left corner of the main menu.
1.7
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
haxelib run lime test html5 -v
## Please describe your issue. Provide extensive detail and images if possible.
Can not compile because of this error and no other issue has gotten a response. Only happens on HTML compile

**Comment by isakube** [comment]
why the fuck are you compiling to html5

oh wait thats the default lime target. let me rephrase that.

why the fuck is the default lime target html5

**Comment by Yuuki795** [comment]
> why the fuck are you compiling to html5
> 
> oh wait thats the default lime target. let me rephrase that.
> 
> why the fuck is the default lime target html5

trust me i have my reasons lol

**Comment by EliteMasterEric** [comment]
> why the fuck are you compiling to html5

Kade Engine is [supposed to support HTML5 builds](https://funkin.puyo.xyz/).

As for this issue, I think I fixed it in #2229 .
